ConfigurationFactory/ConfigurationManager.cs

```diff
@@ -0,0 +1,40 @@
+using System.Text.Json;
+using WeatherConfigurationInterfaces;
```

Comment:
Check this for reading the app configurations in .Net: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-7.0